src/cogent3/core/alignment.py

```diff
@@ -554,6 +554,17 @@ def copy(self):
         result = self.__class__(self, moltype=self.moltype, info=self.info)
         return result
 
+    def deepcopy(self, sliced=True):
+        """Returns deep copy of self."""
+        for seq in self.seqs:
+            try:
+                new_seq = seq.deepcopy(sliced=sliced)
+            except:
+                new_seq = seq.copy()
+        result = self.__class__(self.map, new_seq, moltype=self.moltype, info=self.info.copy())
```

Comment:
this is not right -- it's providing only a single sequence, and the tests must therefore be incorrect.

- Create a dict `new_seqs` which has keys `seq.name` and values `new_seq`.
- Add the argument `force_same_data=True`, to avoid redundant copying
- since this is a deepcopy method, you must use `copy.deepcopy(self.info)`